espnet2/enh/encoder/stft_encoder.py

```diff
@@ -22,6 +22,9 @@ def __init__(
         onesided: bool = True,
         use_builtin_complex: bool = True,
         default_fs: int = 16000,
+        spec_transform_type: str = None,
+        spec_factor: float = 0.15,
+        spec_abs_exponent: float = 0.5,
```

Comment:
Where do these values come from?
Can you explain them?

Reply:
I think they should come from SGMSE https://arxiv.org/pdf/2203.17004.pdf but for spec factor they used 1/3 there

Reply:
if the waveform is standard dev normalized to 1, using 1/3 as spec factor bounds the STFT max value to +- 1.5. 
I think using 0.15 bounds instead to -+ 0.75 which may actually be better. 
Do you normalize the waveform std @LiChenda ?

Reply:
diffusion is super sensitive to input min max range

Reply:
@sw005320 , these numbers come from section V.D in [1]. I added comments to the code. @popcornell , in their journal paper [1], they use 0.15 and 0.5.  I did not normalize the waveform std.

[1] J. Richter, S. Welker, J.-M. Lemercier, B. Lay, and T. Gerkmann, “Speech Enhancement and Dereverberation With Diffusion-Based Generative Models,” IEEE/ACM Transactions on Audio, Speech, and Language Processing, vol. 31, pp. 2351–2364, 2023.

Reply:
Do you normalize by the .abs().amax() to be always within -1 and 1 ? 
https://github.com/sp-uhh/sgmse/blob/c6e3291ee56b07792c9d8c7d7d49487b3042e01b/sgmse/data_module.py#L72

Reply:
I added the normalize option but didn't use it in my model training by default. Because I feel it is not reasonable for causal models. https://github.com/espnet/espnet/blob/12c0eb2c7599cdfdcee22a47f7a4e48c0caab1b8/espnet2/enh/diffusion_enh.py#L113

Reply:
I agree